### PR TITLE
View v1 to v2 conversion

### PR DESCRIPTION
**Pull request by agonza40**
### This project uses [semver](semver.org), please check the scope of this pr:
- [ ] #patch# - backwards-compatible bug fix
- [x] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change
# CHANGELOG

Added utility function to convert version 1 views to version 2 views.

### CODE REVIEW

File: tests/conversion/view-v1-to-v2-test.js [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+    model: 'some',
+    arrayOptions: {
+      itemCell: {
+        extends: 'otherContainer'
+      }
+    }
+  }
+}
+
+const FULL_TEST1 = {
```
Comment (sandersky) [inline tests/conversion/view-v1-to-v2-test.js:65]: Can we move these to separate files so this file doesn't become a monstrous beast. Maybe something along the lines of:

_tests/conversion/fixtures/array.js_

``` js
export const v1 = {…}
export const v2 = {…}
```

_tests/conversion/view-v1-to-v2-test.js_

``` js
import {
  v1 as arrayV1,
  v2 as arrayV2
} from './fixtures/array'
```

## 

You could even go as far as adding an `index.js` in the fixtures file like so:

_tests/conversion/fixtures/index.js_

``` js
import {
  v1 as arrayV1,
  v2 as arrayV2
} from './fixtures/array'

import {
  v1 as complexV1,
  v2 as complexV2
} from './fixtures/complex'

export default [
  {
    v1: arrayV1,
    v2: arrayV2
  },
  {
    v1: complexV1,
    v2: complexV2
  }
]
```

Then in the main test file you can do:

_tests/conversion/view-v1-to-v2-test.js_

``` js
import tests from './fixtures'

…

tests.forEach((test) => {
 …
})
```

**Comment by sandersky** [comment]
At a quick glance I don't think your tests exercise all of the schema, for example you should exercise the various renderers and their custom properties.

**Comment by sandersky** [comment]
👎

**Comment by agonza40** [comment]
@sandersky This PR was essentially to give @cstolli something to work with for the CLI integration. There's more scenarios I'd like to cover with tests but this seemed like a good start.

File: tests/conversion/view-v1-to-v2-test.js (hunk 1 above)
Comment (agonza40) [inline tests/conversion/view-v1-to-v2-test.js:65]: Good idea, I'll set something like this up so that it's using the fixtures.

**Comment by sandersky** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ciena-blueplanet/bunsen-core/pull-request/7/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-blueplanet/bunsen-core)

**Pull request merged**